net/udp-broadcast-relay-redux-openwrt/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udp-broadcast-relay-redux
+PKG_VERSION:=1.0.0
```

Comment:
Is there some more appropriate version than 1.0.0?  The underlying utility doesn't include a version number.

Reply:
```suggestion
```
Okay, just drop this line.
It will be filled automatically.

See also:
https://github.com/openwrt/openwrt/blob/f6cc00650cbdc887a6b7557e6934dec3a04e31da/include/download.mk#L13
```
PKG_VERSION := $(if $(PKG_SOURCE_DATE),$(PKG_SOURCE_DATE)-)$(call version_abbrev,$(PKG_SOURCE_VERSION))
```